packages/jazz-tools/src/tests/groupsAndAccounts.test.ts

```diff
@@ -250,6 +250,10 @@ describe("Group inheritance", () => {
 
     // @ts-expect-error - Even though readerInvite is a valid role for an account, we don't allow it to not create confusion when using the intellisense
     group.addMember(account, "readerInvite");
+    // @ts-expect-error
+    group.addMember(account, "inherit");
+    // @ts-expect-error
+    group.addMember(account, undefined);
```

Comment:
Could you add a brief description of why the type checking should fail?

It would to understand more quickly what these lines are testing.